redis/commands.py

```diff
@@ -1658,6 +1660,14 @@ def xadd(self, name, fields, id='*', maxlen=None, approximate=True,
             if approximate:
                 pieces.append(b'~')
             pieces.append(str(maxlen))
+        if minid is not None:
+            pieces.append(b'MINID')
+            if approximate:
+                pieces.append(b'~')
```

Comment:
IMHO approximate should be False, by default. But, according to the docs it looks like equals (=) or (~) can both be specified explicitly. Can we test case both?

Reply:
I don't understand...
I can't change the default because we don't want to break anything, 
And not specifying the `~` sign is equivalent to adding `=`.

I can change it to:
            ```
if approximate:
                pieces.append(b'~')
            else:
                pieces.append(b'=')
```